main.nf

```diff
@@ -1440,10 +1440,11 @@ process Ascat {
     when: 'ascat' in tools
 
     script:
+    gender = genderMap[idPatient]
     """
     # get rid of "chr" string if there is any
-    for f in *BAF *LogR; do sed 's/chr//g' \$f > tmpFile; mv tmpFile \$f;done
-    Rscript ${workflow.projectDir}/bin/run_ascat.r ${bafTumor} ${logrTumor} ${bafNormal} ${logrNormal} ${idSampleTumor} ${baseDir} ${acLociGC}
+    #for f in *BAF *LogR; do sed 's/chr//g' \$f > tmpFile; mv tmpFile \$f;done
```

Comment:
I can see that you commented this line.
If so, we can probably get rid of it and of the one on top as well?